climada/hazard/tc_tracks.py

```diff
@@ -666,9 +666,9 @@ def equal_timestep(self, time_step_h=1, land_params=False):
                                                       land_geom))
             self.data = new_data
 
-    def calc_random_walk(self, **kwargs):
+    def calc_random_walk(self, ens_size=9, **kwargs):
```

Comment:
I would rename `ens_size` to `nb_synt_tracks` or analog to be clearer about what it stands for.

Reply:
Fine with me (although I find ens_size pretty clear: the size of the generated ensemble, common in climate science but certainly less in other fields), but that may imply breaking some earlier code such as:
`tc_track.calc_random_walk(ens_size=9)`
At the same time since all other input parameters changed this might not be an issue (perhaps even good so users realize the methodology changed), so I'm happy to change to `nb_synth_tracks`.

Reply:
Very good. Any reason why to put default to 9? Maybe there should be no default number of tracks?

Reply:
In my opinion, `nb_synth_tracks` is not very clear either. How about `ensemble_size`?